### PR TITLE
Fix nodal min

### PR DESCRIPTION
**Pull request by thomasgibson**

### CODE REVIEW

**Comment by thomasgibson** [comment]
@inducer I think something in `dagrt` broke the example in `old_symbolic`.

**Comment by inducer** [comment]
Thanks for finding the issue with the `min`. I'll work on getting the dagrt fusion thing fixed up.

**Comment by inducer** [comment]
Dagrt problem is being fixed in #106, which will hopefully merge soon.

**Comment by thomasgibson** [comment]
Thanks @inducer !

File: grudge/op.py [hunk 1]
```diff
@@ -1122,7 +1122,7 @@ def nodal_min(dcoll: DiscretizationCollection, dd, vec):
     # NOTE: Must retain a way to do local reductions
     actx = vec.array_context
     return reduce(lambda acc, grp_ary: actx.np.minimum(acc, actx.np.min(grp_ary)),
-                  vec, -np.inf)
```
Comment (inducer) [inline grudge/op.py:1125]: Wanna add a test? :)

**Review by inducer — APPROVED** [review]
(I approve, test or no.)

File: grudge/op.py (hunk 1 above)
Comment (inducer) [reply grudge/op.py:1125]: (Setting to merge, tests could happen in a separate PR.)

**Pull request merged**